### ISSUE TITLE
It sometimes mistaps!

### ISSUE THREAD

**Issue by myfunvideos** [Fate-Grand-Automata/FGA#1366]
### Preparation

- [X] I tested the [latest release](https://github.com/Fate-Grand-Automata/FGA/releases/tag/canary)
- [X] I looked at other issues (even the [closed ones](https://github.com/Fate-Grand-Automata/FGA/issues?q=is%3Aissue+is%3Aclosed))
- [X] I read the [Troubleshooting Guide](https://github.com/MathewSachin/Fate-Grand-Automata/wiki/Troubleshooting)

### FGO server

NA

### FGA build number

1823

### Describe the bug

So, I decided to download and use fga to try and farm Forbidden Page with my Space Ishtar and make sure to pay attention to the lights moving while I'm not loomg and I think that the program sometimes misclicks a few times because one of the times it clicked the np, but not the two random face cards.

Because of that, it sometimes stops in the middle of it's action and I have to withdraw again to do it again.

Can this be fixed? I don't know the system very well because it's my first time.

I'm a beginner who started using this and would

### Video

I unfortunately do not have enough space at the moment have it recorded.

### Device model

Samsung Galaxy S10e

### Android version

12

### RAM

6GB

**Comment by TheRealPlagiariser** [comment]
When you manage to upload the video, make sure to also send all the settings of your spishtar script, mainly the battle sequence and the card priority.

**Comment by reconman** [comment]
Closing for now until you can provide more details.

**Comment by myfunvideos** [comment]
Found the problem. Turns out, when the game lags and the screen freezes in the middle of an animation, except for the NP being in the middle of use, the script tries to continue the next skill on its commands, but because the screen is in the middle of it being frozen, it doesn't so it misses the command once the screen is unstuck and when it is stuck in the middle of the face cards appearing, it improvises and starts fighting the battle without using the skills or the np. When it wins, it just does the next run as if last battle didn't cause it to improvise. When it loses, it withdraws and tries the right commands.

So in short, wasn't the fga but the game doing its occasional lagging.
